### DISCUSSION TITLE
the length of grains.boundary

### DISCUSSION THREAD

**Discussion by kevindurantnets** [mtex-toolbox/mtex#1990] [Ask Anything]
Dear All,
I'm a material student and beginner of mtex (Matlab as well). The output of 'grains.boundary' in my code don't have length which is shown as follows:
```
 Segments   mineral 1   mineral 2
      373  notIndexed  Forsterite
       12  notIndexed   Enstatite
       10  notIndexed    Diopside
```
and the output in the documentation is shown as follows:
```
Segments    length   mineral 1   mineral 2
      439  27383 µm  notIndexed  Forsterite
       18   1550 µm  notIndexed   Enstatite
       22   1350 µm  notIndexed    Diopside
```
 Could you help me solve this problem?

**Comment by kilir** [comment]
Hi,
for the nice info in µm you need to use a recent version of mtex. The reason for the different numbers of segments should originate from different procedures done on the data I would assume.

Cheers,
Rüdiger
